dist/index.js

```diff
@@ -467,7 +467,7 @@ const core = __webpack_require__(186);
 function checkPrBranch(prBranch) {
   core.info(`Checking branch name formatting\n  - "${prBranch}"`);
 
-  let prBranchRegexp = /^\d+(-[^\W_]+)+$/;
+  let prBranchRegexp = /^\d+([-_][^\W_]+)+$/;
```

Comment:
Is this simultaneously trying to allow (`[-_]`) and disallow (`[^\W_]`) underscores?

Reply:
As I read it, it's effectively allowing underscores as word separators, but not as 'letters' within a word

Reply:
Ok. In my particular case, it's actually a letter within a word but will not be recognized as such. Should I just change the branch name instead of changing the rules?

Reply:
I could change this to `^\d+(-\w+)+$`, to allow `_` within a word, but not as a separator (meaning branch needs to start with a number followed by dash), but then some funny branch will be allowed, for example:
```
1689-add-subphases-to_-lb_iter
1689-add-subphases-to__lb_iter
```
or I can think some more on the proper rule.

Reply:
Maybe `^(?!.*[_-]{2,}.*)\d+(-\w+)+$` would work?

Reply:
In the end, I used `^\d+(-[^\W_]+(_[^\W_]+)*)+$`, so a dash is "main" separator, but within a word an underscore also can be used as a separator (as many times as needed).

Reply:
I think that's reasonable, as long as using an underscore the way I did falls within the spirit of the original restrictions. Either we can make this change or choose not to make any change.